src/main/java/com/google/sps/api/authorization/AuthorizationRequester.java

```diff
@@ -58,19 +58,17 @@
    */
   public static GoogleClientSecrets getClientCredential() throws IOException {
     if (clientSecrets == null) {
+      try{
```

Comment:
1. Please reformat
2. Have you verified that if the file is missing, then the proper message is shown?

Reply:
Yes